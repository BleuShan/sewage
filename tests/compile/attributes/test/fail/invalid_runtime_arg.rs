#[sewage::test(runtime)]
fn run() {}

#[sewage::test(tokio)]
fn run_tokio() {}

fn main() {}
