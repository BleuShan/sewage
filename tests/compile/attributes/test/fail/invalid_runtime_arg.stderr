error: Missing arguments
 --> $DIR/invalid_runtime_arg.rs:1:1
  |
1 | #[sewage::test(runtime)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: Invalid identifier: \"tokio\"
 --> $DIR/invalid_runtime_arg.rs:4:1
  |
4 | #[sewage::test(tokio)]
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
